stripe/_http_client.py

```diff
@@ -1353,6 +1364,113 @@ async def close_async(self):
         await self._client_async.aclose()
 
 
+class AIOHTTPClient(HTTPClient):
+    name = "aiohttp"
+
+    def __init__(
+        self, timeout: Optional[Union[float, "AIOHTTPTimeout"]] = 80, **kwargs
+    ):
+        super(AIOHTTPClient, self).__init__(**kwargs)
+
+        if aiohttp is None:
+            raise ImportError(
+                "Unexpected: tried to initialize AIOHTTPClient but the aiohttp module is not present."
+            )
+
+        self.httpx = httpx
```

Comment:
?

Reply:
Copy pasta removed in [Remove dead code](https://github.com/stripe/stripe-python/pull/1254/commits/385f1f197ee4acf65bd33a868f444506294ad1b4)